### DISCUSSION TITLE
Is there support for @print CSS rules when converting html?

### DISCUSSION THREAD

**Discussion by Typist01** [gotenberg/gotenberg#611] [Q&A]
Hi, wondering if there is a way to hide some elements on the page only when printing like with the @print css media query. And does this functionality remain once the HTML is converted to a PDF using Gotenberg API?

**Comment by gulien** [comment]
Hello @Typist01,

You may select the media print when converting to PDF thanks to the form field:

`emulatedMediaType string  The media type to emulate, either "screen" or "print" - empty means "print"`
